django_python3_ldap/ldap.py

```diff
@@ -208,11 +208,17 @@ def authenticate(*args, **kwargs):
     in settings.LDAP_AUTH_USER_LOOKUP_FIELDS, plus a `password` argument.
     """
     password = kwargs.pop("password", None)
+    ldap_kwargs = {
+        key: value for (key, value) in kwargs.items()
+        if key in frozenset(settings.LDAP_AUTH_USER_LOOKUP_FIELDS)
+    }
+
```

Comment:
You're making a lot of `frozenset(settings.LDAP_AUTH_USER_LOOKUP_FIELDS)`. It would be better to create this at the start of the method, then re-use it.

Reply:
Good point. I submitted a fix.